extensions/tangent/init.lua

```diff
@@ -1695,6 +1690,47 @@ local MESSAGE_SIZE = 1
 -- Message Body.
 local MESSAGE_BODY = 2
 
+-- socketCallback(data, tag) -> none
```

Comment:
`hs.tangent.socketCallback` here. Also I’m not sure what the reasoning is for exposing the callback - is it for people to use as their own Hs.socket callback?

Reply:
It's not exposed - it's just in a local function.